contnumber.py

```diff
@@ -0,0 +1,29 @@
+import csv
+with open('C://22BCA192//PYTHON//contact1.csv','w',newline='') as obj:
+    p=csv.writer(obj)
+    h=['name','contact']
+    p.writerow(h)
+    data=[['om',1234567890],
+          ['sai',1223456789],
+          ['ram',1234455777],
+         ['radha',1234455777],
+         ['gopal',345678902]]
+    p.writerows(data)
+with open('C://22BCA192//PYTHON//contact1.csv','r',newline='') as ob:

```

Comment:
newline=' '? why you can write it?

Reply:
In this program output the compiler added new line by default so we can remove this newline(empty line) we can write 
newline=' '.